### PR TITLE
Github tests log improvement

### PR DESCRIPTION
**Pull request by murrant**
Don't fail when dumping non-existent log file

This pulls focus away from the actual problem

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -191,12 +191,12 @@ jobs:
         name: Dump serve logs
         if: always() && matrix.skip-web-check != '1'
         run: |
-          cat /tmp/artisan-serve.log
+          [ ! -f /tmp/artisan-serve.log ] || cat /tmp/artisan-serve.log
```
Comment (Jellyfrog) [inline .github/workflows/test.yml:194]:
```suggestion
          [ -f /tmp/artisan-serve.log ] && cat /tmp/artisan-serve.log
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -191,12 +191,12 @@ jobs:
...
       -
         name: Dump snmp logs
         if: always() && matrix.skip-unit-check != '1'
         run: |
-          cat /tmp/snmpsimd.log
+          [ ! -f /tmp/snmpsimd.log ] || cat /tmp/snmpsimd.log
```
Comment (Jellyfrog) [inline .github/workflows/test.yml:199]:
```suggestion
          [ -f /tmp/snmpsimd.log ] && cat /tmp/snmpsimd.log
```

File: .github/workflows/test.yml (hunk 1 above)
Comment (Jellyfrog) [reply .github/workflows/test.yml:194]: "if not else", is not as easy to follow :)

Comment (Jellyfrog) [reply .github/workflows/test.yml:194]: `cat /tmp/artisan-serve.log || true`
is even dryer.

Comment (murrant) [reply .github/workflows/test.yml:194]: I like the last one... I started with the second one, but it still returned 1 instead of 0 :D

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**